### PR TITLE
Add Issue and PR Templates

### LINKED ISSUE #256: Add issue forms and pull request template
We need issue forms, bug forms and other templates to facilitate smoother workflow

### PR DESCRIPTION
**Pull request by Zain-Muiz**
Fixes #256 

## Proposed Changes

  - Add Issue Template
  - Add PR Template
  

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/290"></a>

**Note to reviewers** :
After I took up the issue, I think one of the merged PRs, Included ISSUE_TEMPLATE folder inside .github/workflow, On updating my branch before creating this PR, My changes along with the merge that I just mentioned has come to be. Something should be done here before we merge this PR. Do Look at the issue and feel free to suggest any changes.

### CODE REVIEW

File: .github/pull_request.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+## Fixes Issue
+
+This PR fixes the following issues:
+
+#example
+
+<!-- Write down all the changes made-->
+## Changes proposed
+
+Here comes all the changes proposed through this PR
```
Comment (schmelto) [inline .github/pull_request.md:12]:
```suggestion
<!-- Here comes all the changes proposed through this PR -->
```

File: .github/pull_request.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+## Check List (Check all the boxes which are applicable) <!--Follow the above conventions to check the box-->
+
+- [ ] My code follows the code style of this project.
+- [ ] My change requires a change to the documentation.
+- [ ] I have updated the documentation accordingly.
+- [ ] All new and existing tests passed.
+- [ ] This PR does not contain plagiarized content.
+- [ ] The title of my pull request is a short description of the requested changes.
+
```
Comment (schmelto) [inline .github/pull_request.md:28]:
```suggestion
```

File: .github/pull_request.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+<!--Add screen shots of the changed output-->
+## Screenshots 
+Add all the screenshots which support your changes
```
Comment (schmelto) [inline .github/pull_request.md:32]:
```suggestion
<!-- Add all the screenshots which support your changes -->
```

File: .github/pull_request.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+- [ ] My code follows the code style of this project.
+- [ ] My change requires a change to the documentation.
+- [ ] I have updated the documentation accordingly.
+- [ ] All new and existing tests passed.
+- [ ] This PR does not contain plagiarized content.
+- [ ] The title of my pull request is a short description of the requested changes.
+
+
+<!--Add screen shots of the changed output-->
```
Comment (schmelto) [inline .github/pull_request.md:30]:
```suggestion
```

**Review by schmelto — CHANGES_REQUESTED** [review]

**Comment by Zain-Muiz** [comment]
@schmelto Heyy, I have made the proposed changes. Do you want me to squash the commits and push or is it okay otherwise ?

**Comment by schmelto** [comment]
Please push the changes to this PR

**Review by schmelto — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    id: extrainfo
+    attributes:
+      label: Additional information
+      description: Is there anything else we should know about this bugnb?
+    validations:
+      required: false
+  - type: markdown
+    attributes:
+      value: |
+        You can also join the Discord community [here](http://discord.eddiehub.org)
```
Comment (Panquesito7) [inline .github/ISSUE_TEMPLATE/bug.yml:29]:
```suggestion
        You can also join our Discord community [here](http://discord.eddiehub.org)
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 6]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+- name: 🙋🏾🙋🏼‍Question
+  url: http://discord.eddiehub.org/
+  about: Feel free to ask your question on our discord channel.
```
Comment (Panquesito7) [inline .github/ISSUE_TEMPLATE/config.yml:5]:
```suggestion
  about: Feel free to ask your question on our Discord channel.
```

File: .github/ISSUE_TEMPLATE/docs.yml [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+    id: extrainfo
+    attributes:
+      label: Additional information
+      description: Is there anything else we should know about this issue?
+    validations:
+      required: false
+  - type: markdown
+    attributes:
+      value: |
+        You can also join the Discord community [here](http://discord.eddiehub.org)
```
Comment (Panquesito7) [inline .github/ISSUE_TEMPLATE/docs.yml:30]:
```suggestion
        You can also join our Discord community [here](http://discord.eddiehub.org)
```

File: .github/ISSUE_TEMPLATE/other.yml [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+      description: Provide a clear and concise explanation of your issue.
+    validations:
+      required: true
+  - type: textarea
+    id: extrainfo
+    attributes:
+      label: Additional information
+      description: Is there anything else we should know about this issue?
+    validations:
+      required: false
```
Comment (Panquesito7) [inline .github/ISSUE_TEMPLATE/other.yml:22]:
```suggestion
      required: false
  - type: markdown
    attributes:
      value: |
        You can also join the Discord community [here](http://discord.eddiehub.org)
        Feel free to check out other cool repositories of the EddieHub Community [here](https://github.com/EddieHubCommunity)
```

**Pull request merged**